### PR TITLE
feat(ai-monitoring): Add global config for AI model costs

### PR DESCRIPTION
**Pull request by colin-sentry**
Context is here: https://github.com/getsentry/relay/pull/3554#pullrequestreview-2045384821

### CODE REVIEW

File: src/sentry/relay/config/ai_model_costs.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+def ai_model_costs_config() -> AIModelCosts:
+    return {
+        "version": 1,
+        "costs": list(
+            map(
+                lambda row: {
+                    "modelId": row[0],
+                    "forCompletion": row[1],
+                    "costPer1kTokens": row[2],
+                },
```
Comment (jjbayer) [inline src/sentry/relay/config/ai_model_costs.py:24]:
```suggestion
        "costs": nit

```suggestion
        "costs": [
            {
                "modelId": row[0],
                "forCompletion": row[1],
                "costPer1kTokens": row[2],
            }
            for row in [
```
```

**Review by jjbayer — APPROVED** [review]
I'll trigger a relay version bump to make the tests pass.

File: requirements-base.txt [hunk 2]
```diff
@@ -66,7 +66,7 @@ sentry-arroyo>=2.16.5
 sentry-kafka-schemas>=0.1.81
 sentry-ophio==0.2.7
 sentry-redis-tools>=0.1.7
-sentry-relay>=0.8.60
+sentry-relay>=0.8.63
```
Comment (jjbayer) [inline requirements-base.txt:69]:
```suggestion
sentry-relay>=0.8.64
```

**Review by asottile-sentry — APPROVED** [review]

**Review by Dav1dde — APPROVED** [review]

**Pull request merged**